.nengobones.yml

```diff
@@ -61,11 +61,25 @@ ci_scripts:
     output_name: remote-docs
     host: azure-docs
     travis_var_key: 2895d60e3414
+    azure_name: nengo-dl-docs
+    azure_group: nengo-ci
+    remote_vars:
+      NUMPY_VERSION: $NUMPY_VERSION
+      TF_VERSION: $TF_VERSION
+    remote_setup:
+      - conda install -y cudatoolkit=10.1 cudnn
```

Comment:
It's interesting that this works for TF 2.4, which requires CUDA 11. I'm guessing that on our Azure machine, we've got CUDA 11 installed globally, so it's not even using this CUDA for TF 2.4. I'll look to see if we can remove it.

Reply:
So looking at the docs build, where we can actually see the TF printouts when it's starting up, it turns out that we actually weren't loading the CUDA 11 libraries, and thus weren't using the GPU.

I changed this to install CUDA 11, and it also turns out that it's slower when we do use the GPU. I'm not sure why exactly; potentially a combination of longer install times for CUDA 11, and the models being small and thus slower on the GPU.

Without GPU: https://travis-ci.com/github/nengo/keras-spiking/builds/214122937
With GPU: https://travis-ci.com/github/nengo/keras-spiking/builds/214129937

Anyway, I think it's good to test against GPU, since we're not doing that in our tests that run on TravisCI servers, so I'll keep our remote tests running on GPU with CUDA 11, even though it's slower.